docs/deployment/managed-scanning/azure.md

```diff
@@ -0,0 +1,121 @@
+---
+slug: azure
+append_help_link: true
+title: Azure DevOps
+hide_title: true
+description: Learn how to add an Azure DevOps repository to Semgrep Managed Scans.
+tags:
+  - Beta
+  - Deployment
+  - Semgrep AppSec Platform
+---
+
+import ScanWithSms from "/src/components/procedure/_scan-with-sms.mdx"
+
+# Add an Azure DevOps repository to Semgrep Managed Scans
+
+Add Azure DevOps repositories to your Semgrep organization in bulk without adding or changing your existing CI workflows through **Managed Scans**. 
+
+
+## Prerequisites and permissions
+
+Semgrep Managed Scanning requires one of the following plans:
+
+//TODO
```

Comment:
Which ADO plans are supported? Do we just say we support only ADO Cloud?

Reply:
Yeah, we can just say ADO Cloud, this will work whether you have basic or test plans: https://azure.microsoft.com/en-us/pricing/details/devops/azure-devops-services/